src/liballoc/rc.rs

```diff
@@ -1830,3 +1830,6 @@ impl<T: ?Sized> AsRef<T> for Rc<T> {
         &**self
     }
 }
+
+#[unstable(feature = "pin", issue = "49150")]
+impl<T: ?Sized> Unpin for Box<T> { }
```

Comment:
This looks like a copy-paste woops, it should be `Rc`, right? XD

Reply:
yep whoops!